cmake/core-files.cmake

```diff
@@ -444,13 +444,16 @@ set(MBGL_CORE_FILES
     include/mbgl/util/geo.hpp
     include/mbgl/util/geojson.hpp
     include/mbgl/util/geometry.hpp
+    include/mbgl/util/ignore.hpp
     include/mbgl/util/image.hpp
+    include/mbgl/util/interpolate.hpp
     include/mbgl/util/logging.hpp
     include/mbgl/util/noncopyable.hpp
     include/mbgl/util/optional.hpp
     include/mbgl/util/platform.hpp
     include/mbgl/util/projection.hpp
     include/mbgl/util/range.hpp
+    include/mbgl/util/rapidjson.hpp
```

Comment:
We could make `Result<GeoJSON> convertGeoJSON(const QVariant& value)` to use an overloaded version of `convertGeoJSON` internally that receives a `std::string` as input. This overloaded version could then use _rapidjson_ to parse this and reside outside of Qt scope.

The same could be made in e.g. https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/src/style/conversion/geojson.hpp.